OrangeDesignSystemDemo/OrangeDesignSystemDemo/Screens/Components/Pages/Lists/ListItemVariant/LeadingOption.swift

```diff
@@ -24,13 +24,16 @@ enum LeadingOption: Int, CaseIterable {
     case circle
     case wide
     case square
+    case appIcon = 10 // To be shure is the last in chips list
```

Comment:
Je pense qu'il serait plus cohérent de donner des valeurs entières à chaque _case_ de sorte àc e que cette valeur 10 ne sorte pas de nul part.

Il faudrait refaire le commentaire, _shure_ n'exichte pas, che n'est pas un mot in english :D